langgraph/src/pregel/write.ts

```diff
@@ -92,6 +93,15 @@ export class ChannelWrite<
       Object.entries(values).filter(([_channel, value]) => value !== SKIP_WRITE)
     );
   }
+
+  static isWriter(runnable: RunnableLike): boolean {
+    // eslint-disable-next-line no-instanceof/no-instanceof
+    return runnable instanceof ChannelWrite;
```

Comment:
Double check: Currently used in `PregelNode.pipe()`. Need to double check that this is the desired implementation.